brayns/parameters/GeometryParameters.cpp

```diff
@@ -211,10 +212,20 @@ void GeometryParameters::parse(const po::variables_map& vm)
     {
         _colorScheme = ColorScheme::none;
         const auto& colorScheme = vm[PARAM_COLOR_SCHEME].as<std::string>();
-        for (size_t i = 0; i < sizeof(COLOR_SCHEMES) / sizeof(COLOR_SCHEMES[0]);
-             ++i)
-            if (colorScheme == COLOR_SCHEMES[i])
-                _colorScheme = static_cast<ColorScheme>(i);
+        if (!colorScheme.empty())
+        {
+            bool matchedScheme = false;
+            for (size_t i = 0; i < COLOR_SCHEMES.size(); i++)
+                if (colorScheme == COLOR_SCHEMES[i])
+                {
+                    _colorScheme = static_cast<ColorScheme>(i);
+                    matchedScheme = true;
+                    break;
+                }
+
+            if (!matchedScheme)
+                throw po::error("No match for color scheme '" + colorScheme);
+        }
```

Comment:
Does it look shorter with std::find? (in fact, that would be a reason for using std::array)

Reply:
done